tests/test_endpoint_on_development.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -uo pipefail
+
+source tests/test_endpoint_common.sh
+
+# Download Legato and build the tools
+repo init -u git://github.com/legatoproject/manifest -m legato/releases/${LEGATO_VERSION}.xml
```

Comment:
Don't check out Legato AF source code in test script! Integrate the process in prebuilt/preconfigured environments.

Reply:
We always expect the ways to shorten testing time.